src/faker.ts

```diff
@@ -173,6 +175,10 @@ export class Faker extends SimpleFaker {
    * @param options.locale The name of the main locale to use.
    * @param options.localeFallback The name of the fallback locale to use.
    *
+   * @example Why are you looking at examples here? It's deprecated!
```

Comment:
I wont like to support passive aggressive phrases in our docs :thinking:

Reply:
Then please provide a working example.

Reply:
the absolut minimum would be:
```suggestion
   * @example new Faker({ locales });
```